src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,19 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
+    private static final int NUMBER_OF_FIGURES = 6;
```

Comment:
The `NUMBER_OF_FIGURES` constant is fine, but remember that static methods are generally discouraged. If `FigureSupplier` methods are not static, consider creating an instance of `FigureSupplier` outside of the `main` method and use it within the method.